apps/condo/domains/user/schema/User.test.js

```diff
@@ -735,4 +738,58 @@ describe('Sensitive data search', () => {
             }, ['meta'])
         })
     })
+    test('Email address of service users can be read by another service users with direct access', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [rightsSet] = await createTestUserRightsSet(admin, {
+            canReadUsers: true,
+            canReadUserEmailField: true,
+        })
+        const firstServiceUser = await makeClientWithServiceUser({
+            rightsSet: { connect: { id: rightsSet.id } },
+        })
+        const secondServiceUser = await makeClientWithServiceUser()
+        const nonServiceUser = await makeClientWithNewRegisteredAndLoggedInUser()
+
+        const UserWithEmail = generateGQLTestUtils(generateGqlQueries('User', '{ id email deletedAt }'))
```

Comment:
IMO this is strange behaviour of codegen functions usage inside tests. Why you can't reuse test utils from user domain?

Reply:
1. `User` is not an option, since it does not contain `email` field
2. `UserAdmin` is not an option, since it contains `phone` field, and I'll get `AccessDeniedError` on each request, which is hard to process in tests.

So I've created my selection of subfields